cmd/kubeless/update.go

```diff
@@ -125,4 +135,6 @@ func init() {
 	updateCmd.Flags().StringP("schedule", "", "", "Specify schedule in cron format for scheduled function")
 	updateCmd.Flags().Bool("trigger-http", false, "Deploy a http-based function to Kubeless")
 	updateCmd.Flags().StringP("runtime-image", "", "", "Custom runtime image")
+	updateCmd.Flags().Bool("headless", false, "Deploy the function as a headless service")
```

Comment:
I would disable the `headless` parameter for the update command or make a function that actually checks if the value has been set (it is doable going over the provided parameters with the `flag.Visit` function). Since it is a boolean, it is difficult to say if the user wants to update its value to `false` or if the flag is missing.
Also I would set here the default value of the port to `-1` or something like that to differentiate the intention of the user (and not use it if the value is the default).